install/configuring_inventory_file.adoc

```diff
@@ -455,12 +455,13 @@ Groups and Host Mappings] for the current method of setting node labels.
 a|This variable configures additional `docker` options in
 *_/etc/sysconfig/docker_*, such as options used in
 xref:host_preparation.adoc#managing-docker-container-logs[Managing Container Logs].
-It is recommended to use `json-file`.
+We recommend that you use `json-file`.
 
 The following example shows the configuration of Docker to use the `json-file` log
-driver, where Docker will rotate between three 1 MB log files:
+driver, where Docker can rotate between three 1 MB log files, and signature verification is not required.
+When supplying additional options, be sure you maintain the single quotation mark formatting:
```

Comment:
I think we use _ensure_ over _be sure_ by a margin 1780-to-5 based on an Atom search. 
_ensure you maintain_ ??

Reply:
I'd s/be sure you/ensure that you

Reply:
HAHAHA!! OK, I can live with that! :-)

Reply:
DONE